src/main/java/eu/europeana/downloads/ListRecordsQuery.java

```diff
@@ -29,54 +29,42 @@
     private static final String ZIP_EXTENSION = ".zip";
     private static final String PATH_SEPERATOR = "/";
 
-    @Value("${ListRecords.metadataPrefix}")
+    @Value("${metadataPrefix}")
     private String metadataPrefix;
 
-    @Value("${ListRecords.from}")
+    @Value("${harvest-from}")
     private String from;
 
-    @Value("${ListRecords.until}")
-    private String until;
-
-    @Value("${ListRecords.set}")
+    @Value("${harvest-sets}")
     private String set;
 
-    @Value("${ListRecords.useGetRecord}")
-    private boolean useGetRecord;
-
-    @Value("${ListRecords.threads}")
-    private int threads;
-
-    private ExecutorService threadPool;
-
-    @Value("${LogProgress.interval}")
+    @Value("${log-progress-interval}")
     private Integer logProgressInterval;
 
-    @Value("${saveToFile}")
-    private String saveToFile;
-
-    @Value("${saveToFolder}")
+    @Value("${sets-folder}")
     private String directoryLocation;
 
+    @Value("${harvest-threads}")
+    private int threads;
 
     private List<String> sets = new ArrayList<>();
 
+    private ExecutorService threadPool;
+
     public ListRecordsQuery() {
     }
 
-    public ListRecordsQuery(String metadataPrefix, String set, String directoryLocation, String saveToFile, int logProgressInterval) {
+    public ListRecordsQuery(String metadataPrefix, String set, String directoryLocation, int logProgressInterval) {
         this.metadataPrefix = metadataPrefix;
         this.set = set;
         this.directoryLocation = directoryLocation;
-        this.saveToFile = saveToFile;
         this.logProgressInterval = logProgressInterval;
         initSets();
```

Comment:
SonarQube doesn't like us calling an overridable method in the constructor (see http://sonarqube.eanadev.org:9000/project/issues?id=eu.europeana%3Adownloads&issues=AXBTOhr5AlJAzPS_E8AC&open=AXBTOhr5AlJAzPS_E8AC) 
I propose to make the initsets method final.